external-import/sentinel-incidents/src/sentinel_incidents_connector/connector.py

```diff
@@ -157,19 +188,78 @@ def _extract_intelligence(self, incident) -> list[object]:
                             )
                         )
                         stix_objects.append(stix_relationship_hostname)
-                    case "#microsoft.graph.security.processEvidence":
-                        stix_file = self.converter_to_stix.create_evidence_file(
-                            evidence
+
+                    # processEvidence, fileEvidence and fileHashEvidence
+                    case evidence_file if evidence_file in priority_evidence_files:
```

Comment:
I think this would be clearer to indicate you are looking in keys
```suggestion
                    case evidence_file if evidence_file in priority_evidence_files.keys():
```